gossip/privdata/reconcile.go

```diff
@@ -120,7 +120,6 @@ func (r *Reconciler) run() {
 			r.logger.Debug("Start reconcile missing private info")
 			if err := r.reconcile(); err != nil {
 				r.logger.Error("Failed to reconcile missing private info, error: ", err.Error())
-				break
```

Comment:
If the (misguided in this case) intent was to break out of the for loop, shouldn't we add the code to actually do so rather than just removing this ineffectual break? @yacovm

Reply:
I [don't think](https://github.com/hyperledger/fabric/commit/fefb5a772e5607363df07516827a64e8e966563a) the attempt was to break out of the for.

Reply:
I should have provided more information to make it easier to review.The for loop is [only start once](https://github.com/hyperledger/fabric/blob/master/gossip/privdata/reconcile.go#L108) when [InitializeChannel](https://github.com/hyperledger/fabric/blob/master/gossip/service/gossip_service.go#L376),so i think an error should not stop the entire function (for the next try or get other private data) . What's more the commit yacovm provide show more clearly that the purpose is to prevent the execution of the following statement.But now it is the last statement.

Reply:
in this case intent was not to break out of the for loop, but rather from select, cause reconciliation is a background process triggered periodically. As @Taction pointed out break Is the last statement here hence could be removed as doesn't add anything to the code flow.